.github/workflows/ai-lab-e2e-nightly-windows.yaml

```diff
@@ -186,7 +186,7 @@ jobs:
           -e OUTPUT_FOLDER=/data \
           -e DEBUG=true \
           -v $PWD:/data:z \
-          quay.io/odockal/pde2e-podman:v0.0.1-windows  \
+          quay.io/odockal/pde2e-podman:v0.0.2-windows  \
             pd-e2e/podman.ps1 \
               -downloadUrl ${{ env.PODMAN_URL }} \
               -version ${{ env.PODMAN_VERSION }} \
```

Comment:
Switching to 0.0.2 might also mean that you need to remove the version parameter.

Reply:
Which wouild mean also removing the podman version paramters from the whole file - inputs and default env. vars.

Reply:
I believe it's fixed now